app/cdap/components/AddTransformation/SelectColumnsList/index.tsx

```diff
@@ -0,0 +1,247 @@
+            <Typography className={classes.subHeaderMessage}>
+              {T.translate('features.WranglerNewSelectCoulmnList.noMatchColumnDatatype')}
+            </Typography>
+          </Box>
+        </Box>
+      ) : (
+        <TableContainer component={Box}>
+          <Table aria-label="recipe steps table" className={classes.tabledisplayStyles}>
+            <TableHead>
+              <TableRow className={`${classes.recipeStepsTableRowStyles} ${classes.rowsOfTable}`}>
+                <TableCell
+                  classes={{
+                    head: `${classes.recipeStepsTableHeadStyles} ${classes.columnstyles}`,
+                  }}
+                ></TableCell>
+                <TableCell
+                  classes={{
+                    head: `${classes.recipeStepsTableHeadStyles} ${classes.nullValueHead}`,
+                  }}
+                >
+                  {T.translate('features.WranglerNewAddTransformation.columns')}
+                </TableCell>
+                <TableCell
+                  classes={{
+                    head: `${classes.recipeStepsTableHeadStyles} ${classes.nullValueHead}`,
+                  }}
+                >
+                  {T.translate('features.WranglerNewAddTransformation.nullValues')}
+                </TableCell>
+              </TableRow>
+            </TableHead>
+            <TableBody>
+              {columns.map((eachColumn, index) => {
+                if (directiveFunctionSupportedDataType.includes('all')) {
+                  return (
+                    <TableRow
+                      className={`${classes.recipeStepsTableRowStyles} ${classes.rowsOfTable}`}
+                      key={index}
+                    >
+                      <TableCell
+                        classes={{
+                          body: `${classes.recipeStepsTableRowStyles} ${classes.radioButtonCellStyles}`,
+                        }}
+                      >
+                        <Radio
+                          color="primary"
+                          onChange={(e) => onSelect(eachColumn)}
+                          checked={
+                            selectedColumns.filter((el) => el.label == eachColumn.label).length
+                              ? true
+                              : false
+                          }
+                        />
+                      </TableCell>
+                      <TableCell classes={{ body: classes.recipeStepsTableRowStyles }}>
+                        <Typography className={classes.recipeStepsActionTypeStyles}>
+                          {eachColumn.label}
+                        </Typography>
+                        <Typography className={classes.recipeStepsActionTypeStyles}>
+                          {eachColumn.type}
+                        </Typography>
+                      </TableCell>
+                      <TableCell
+                        className={[
+                          classes.recipeStepsTableRowStyles,
```

Comment:
we can use string interpolation here
{`${class1} ${class2}`}

Reply:
make the same change everywhere with 2 or more classes in ur code changes and going forward

Reply:
Done